README.md

```diff
@@ -10,28 +10,30 @@ Clone FoodAdvisor
 git clone https://github.com/strapi/foodadvisor.git
 ```
 
-## 2. Start Strapi from the ./api folder
+Navigate to your *./foodadvisor* folder by running `cd foodadvisor` from your command line.
 
-`Path: ./my-projects/foodadvisor/api`:
+## 2. Start Strapi
 
-Run the following from your command line:
+Naviagte to your *./api folder* by running `cd api` from your command line.
+
+In `./my-projects/foodadvisor/api`, run the following from your command line:
 
 ```
-cd api && yarn && yarn run seed && yarn develop
+yarn && yarn run seed && yarn develop
 ```
 
 Then, follow [Step 3. Create an admin user](https://strapi.io/documentation/3.0.0-beta.x/getting-started/quick-start-tutorial.html#_3-create-an-admin-user) to create an `Administrator`.
```

Comment:
I know that it's not an update you did, but can you please update the link here to redirect to the last version of the docs instead of the beta.
Worth it to mention that it's also **step 2** instead of **step 3**.
Here is the doc link for the stable: https://strapi.io/documentation/v3.x/getting-started/quick-start.html#_2-create-an-administrator-user